### DISCUSSION TITLE
Reduce binary size for Pocketbase as a framework.

### DISCUSSION THREAD

**Discussion by HugoOlsson** [pocketbase/pocketbase#3851] [Q&A] [answered]
When I use Pocketbase as a framework, the binary size of the compiled executable is around 50 MB without any added code. Is there any way to reduce this? Like for example to disconnect features that I will not use.

**Comment by ganigeorgiev** [comment, marked as answer]
Please check https://github.com/pocketbase/pocketbase/discussions/3665.

There is no way at the moment to reduce the binary size without removing the `aws-sdk-go` dependency.

**Reply by ganigeorgiev** [reply]
_Side-note: you can specify the `-s -w` linker options to disable the symbol table and DRAWF that should help shrink the result binary a little, eg.: `go build -ldflags="-s -w"`_

**Reply by dschissler** [reply]
Getting the size down will really help with compiling to Raspberry Pi.  If the binary size was 50% and with a 2x speed increase from 4B to 5 then it should only take about 5 seconds to recompile.
